src/startPlugin.js

```diff
@@ -0,0 +1,34 @@
+import pluginApp from './reducers';
+import { startSocket } from './socket';
+import startErrorHandling from './startErrorHandling';
+import startReloader from './startReloader';
+import handlePolling from './handlePolling';
+import { authenticate, setPollUrl } from './actions';
+import { createStore } from 'redux';
+import { Socket } from 'phoenix';
+
+const startPlugin = ({ auth, pollUrl, enhancer, reloader, socketConstructor = Socket }) => {
+  const store = createStore(pluginApp, enhancer);
+
+  startErrorHandling(store);
+
+  if (reloader) {
+    startReloader(store, reloader);
+  }
+
+  if (auth) {
+    store.dispatch(authenticate(auth));
+  }
+
+  if (pollUrl) {
+    store.dispatch(setPollUrl(pollUrl));
+  }
```

Comment:
If auth is null then the socket won't connect until the `authenticate` action is dispatched. If pollUrl is null then polling won't do anything until `setPollUrl` is dispatched. Both can be dispatched when the tester logs into their portal page (see chrom/listenAuth.js).